subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/InstantExecutionEnablingIntegrationTest.groovy

```diff
@@ -75,5 +82,12 @@ class InstantExecutionEnablingIntegrationTest extends AbstractInstantExecutionIn
 
         then: 'it does not enable instant execution'
         fixture.assertNoInstantExecution()
+
+        cleanup:
+        if (previousProp != null) {
```

Comment:
I see the same snippet is used in the test above so maybe extract a `setOrClearProperty`?

Reply:
Done